src/codec_dav1d.c

```diff
@@ -57,8 +57,13 @@ static avifBool dav1dCodecGetNextImage(struct avifCodec * codec,
 {
     if (codec->internal->dav1dContext == NULL) {
         // Give all available threads to decode a single frame as fast as possible
+#if DAV1D_API_VERSION_MAJOR >= 6
```

Comment:
Victorien: I have a question for you. If the max_frame_delay and n_threads are renamed before the next dav1d release, will DAV1D_API_VERSION_MAJOR be incremented?

Reply:
Before the next release? hmmm I don't think so, this patch isn't part of a release yet, this will be in for 1.0.0, so I don't think we need to bump major several times between releases... maybe ask on the irc channel to be sure
Tho I don't see why we would rename those as of now.

Reply:
In case it helps, I'm happy to make another patch if Dav1d's API changes again.